public/app/percona/shared/components/PerconaBootstrapper/PerconaUpdateVersion/PerconaUpdateVersion.test.tsx

```diff
@@ -40,6 +40,31 @@ describe('PerconaUpdateVersion', () => {
     jest.clearAllMocks();
   });
 
+  it('should render modal without changelogs', async () => {
+    const changeLogsAPIResponse = {
+      last_check: '',
+      updates: []
+    };
+    const state = {
+      updates: {
+        isLoading: false,
+        updateAvailable: true,
+        latest: { version: '3.0.1' },
+        lastChecked: '',
+        showUpdateModal: true,
+      },
+    };
+    jest.spyOn(UpdatesService, 'getUpdatesChangelogs').mockReturnValue(Promise.resolve({ ...changeLogsAPIResponse }));
```

Comment:
I think you can just past the changelogsApiResponse directly (no need to spread into a new object)